Import/SampleForLabelsInReports/Import/Report/Part Report With Label.xml

```diff
@@ -9,100 +9,101 @@
-            </td>
-	  </tr>
-          <tr valign="top">
-            <td class="cellHeader" uniqueID="ms__id81" align="left" colspan="" rowspan="" height="" width="" style="border-left:1px #666666 solid;">Part Number</td>
-            <td class="cellHeader" uniqueID="ms__id82">Name</td>
-            <td class="cellHeader" uniqueID="ms__id83">Cost</td>
-            <td class="cellHeader" uniqueID="ms__id84">Make/Buy</td>
-            <td class="cellHeader" uniqueID="ms__id84">Make/Buy Label</td>
-	  </tr>
-          <xsl:for-each select="//Item">
-            <xsl:sort select="name" order="ascending" case-order="upper-first" data-type="text"></xsl:sort>
-            <tr valign="center">
-              <td style="font-family:helvetica;font-size:8pt;font-weight:bold;border-left:1px #666666 solid;padding:2px;" class="cellSolidRightBottom" align="left" uniqueID="ms__id90" colspan="" rowspan="" height="" width="">
-                <xsl:value-of select="item_number"></xsl:value-of>
-                <xsl:if test="item_number='' or not(item_number)">
-                  <xsl:text> </xsl:text>
-	        </xsl:if>
-	      </td>
-              <td style="font-family:helvetica;font-size:8pt;padding:2px;" class="cellSolidRightBottom" align="left" uniqueID="ms__id91">
-                <xsl:value-of select="name"></xsl:value-of>
-                <xsl:if test="name='' or not(name)">
-                  <xsl:text> </xsl:text>
-	        </xsl:if>
-	      </td>
-              <td style="font-family:helvetica;font-size:8pt;padding:2px;" class="cellSolidRightBottom" align="left" uniqueID="ms__id92">
-                <xsl:value-of select="cost"></xsl:value-of>
-                <xsl:if test="cost='' or not(cost)">
-                  <xsl:text> </xsl:text>
-	        </xsl:if>
-	      </td>
-              <td style="font-family:helvetica;font-size:8pt;padding:2px;" class="cellSolidRightBottom" align="left" uniqueID="ms__id93">
-                <xsl:value-of select="make_buy"></xsl:value-of>
-                <xsl:if test="make_buy='' or not(make_buy)">
-                  <xsl:text> </xsl:text>
-	        </xsl:if>
-	      </td>
-              <td style="font-family:helvetica;font-size:8pt;padding:2px;" class="cellSolidRightBottom" align="left" uniqueID="ms__id93">
-                <xsl:value-of select="partMakeBuyLabel"></xsl:value-of>
-                <xsl:if test="partMakeBuyLabel='' or not(partMakeBuyLabel)">
-                  <xsl:text> </xsl:text>
-	        </xsl:if>
-	      </td>
-	     </tr>
-	   </xsl:for-each>
-	  </table>
-	 </body>
-      <script src="../../javascript/PopupMenu.js"></script>
-    </html>
+  <xsl_stylesheet><![CDATA[<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:msxsl="urn:schemas-microsoft-com:xslt" xmlns:user="urn:user-scripts" xmlns:aras="http://www.aras-corp.com" version="1.0">
```

Comment:
The XSL in this report needs to be properly formatted. Otherwise, it's very difficult to read and understand what's happening at a glance.